sdk/digitaltwins/Azure.DigitalTwins.Core/src/RequestOptions.cs

```diff
@@ -13,13 +11,16 @@ namespace Azure.DigitalTwins.Core
     public class RequestOptions
     {
         /// <summary>
-        /// A string representing a weak ETag for the entity that this request performs an operation against, as per RFC7232. The request's operation is performed
+        /// An Etag object for the entity that this request performs an operation against, as per RFC7232. The request's operation is performed
         /// only if this ETag matches the value maintained by the server, indicating that the entity has not been modified since it was last retrieved.
-        /// To force the operation to execute only if the entity exists, set the ETag to the wildcard character '*'. To force the operation to execute unconditionally, leave this value null.
+        /// To force the operation to execute only if the entity exists, set the ETag to the wildcard character '*'. <code>new Etag("*")</code>. To force the operation to execute unconditionally, leave this value null.
         /// </summary>
+        /// <see href="https://docs.microsoft.com/en-us/dotnet/api/azure.etag?view=azure-dotnet">
```

Comment:
See is not a standalone tag. It must be nested in remarks or summary.

We don't want summary to get too big, so really much of that should be moved to remarks. You can then put this bit referencing that URL in remarks.

Reply:
Also, when you get that much stuff in a remarks section, I think what we want to do is separate different thoughts with the `<para></para>` tags.